itest/rust/src/transform3d_test.rs

```diff
@@ -0,0 +1,75 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+use crate::{assert_inner_equivalent, itest};
+
+use godot::prelude::{inner::InnerTransform3D, *};
+
+const TEST_TRANSFORM: Transform3D = Transform3D::new(
+    Basis::from_cols(
+        Vector3::new(1.0, 2.0, 3.0),
+        Vector3::new(4.0, 5.0, 6.0),
+        Vector3::new(7.0, 8.0, -9.0),
+    ),
+    Vector3::new(10.0, 11.0, 12.0),
+);
+
+pub(crate) fn run() -> bool {
+    let mut ok = true;
+    ok &= transform3d_equiv();
+    ok
+}
+
+#[itest]
+fn transform3d_equiv() {
+    assert_inner_equivalent!(InnerTransform3D, TEST_TRANSFORM.affine_inverse(), |a, b| {
+        Transform3D::is_equal_approx(&a, &b)
+    });
+    assert_inner_equivalent!(
+        InnerTransform3D,
+        TEST_TRANSFORM.orthonormalized(),
+        |a, b| { Transform3D::is_equal_approx(&a, &b) }
```

Comment:
Shouldn't need a `{ }`, as it's only one expression.
Many other places as well.

Reply:
you're right, i think it got automatically added for some reason and then i copied it around a whole bunch